backend/model/Users.js

```diff
@@ -84,6 +84,16 @@ const UsersAPI = {
     return userProfile;
   },
 
+  checkUserExists: async function(email) {
+    let success = false;
+    const userRef = usersCollection.where('email', '==', email);
+    const user = await userRef.get();
+    if (!user.exists) {
+      success = true;
+    }
+    return { out: status };
+  },
+
```

Comment:
Good function. Need to update documentation

Reply:
Do we have an email field?

Reply:
Oh oops we're not storing the email so this should be with something else.. maybe use authentication? I can work on this if you need

Reply:
I take it back. Bad function. ;) jk